### PR TITLE
MRG: rewrite readme: name of supported models, link to doc, update bib

### LINKED ISSUE #74: Rewrite readme
I think several of us think we should reshape the readme, with, at least:
- [x] clearer link to the doc
- [x] I think that thank to @agramfort there is no need to install the full conda env now, `pip install -e .` should work
- [x] I was thinking of some images of the enet optimization for the figure, maybe with a gif? wdyt?

### PR DESCRIPTION
**Pull request by mathurinm**
closes #74

### CODE REVIEW

**Comment by mathurinm** [comment]
@agramfort anything missing ?

File: README.rst [hunk 1]
```diff
@@ -3,112 +3,51 @@ sparse-ho
 
 |image0| |image1|
 
-sparse-ho stands for "sparse hyperparameter optimization". This code aims to offer
-a efficient solution to the problem of hyperparameter setting for sparse models such as Lasso.
-Contrarily to grid search, the proposed method can handle efficiently many hyperparameters.
+`sparse-ho` stands for "sparse hyperparameter optimization".
+This package implement efficient hyperparameter tuning for sparse machine learning models.
```
Comment (agramfort) [inline README.rst:7]:
```suggestion
This package implements efficient hyperparameter tuning for sparse machine learning models.
```

File: README.rst [hunk 2]
```diff
@@ -3,112 +3,51 @@ sparse-ho
...
+In order to benchmark performances, the package also implements alternatives such as forward or backward differentiation.
 
+Documentation
+-------------
 
-Install
--------
-
-To be able to run the experiments  you should install the conda environment:
+Please visit https://qb3.github.io/sparse-ho for the latest version of the documentation.
```
Comment (agramfort) [inline README.rst:17]:
```suggestion
Please visit `https://qb3.github.io/sparse-ho`_ for the latest version of the documentation.
```

**Review by agramfort — APPROVED** [review]

**Comment by agramfort** [comment]
thx @mathurinm

**Pull request merged**